README.md

```diff
@@ -7,20 +7,182 @@ This module consists of information about the artist and track as well as the ab
+| ---- | ---- | ----------- |
+| 'comment_id' | 'int' | _Required_. Identifier for comment. |
+| 'song_id' | 'int' | _Required_. Identifier for which song this comment belongs to. |
+| 'user_id' | 'int' | _Required_. Identify which user wrote the comment. |
+| 'content' | 'str' | _Required_. Comment content. |
+| 'time' | 'int' | _Required_. Timestamp for comment. |
+| 'track_time' | 'int' | _Reuiqred_. Track time stamp for comment. |
+
+
+### GET
+
+#### GET /song/:song_id
+Get the song with id. Returns a JSON object.
+
+**Parameters**
+
+| Name  | Type | Description |
+| ----- | ---- | ----------- |
+| 'song_id' | 'int' | _Required_. Identifier for the song. |
+
+**Response**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| 'song_id' | 'int' | _Required_. Identifier for song. |
+| 'song_name' | 'str' | _Required_. Name of song. |
+| 'artist_id' | 'int' | _Required_. Identifier for song artist. |
+| 'release_date' | 'str' | _Required_. Release date of song. |
+| 'P-line' | 'str' | _Required_. Owner to rights of sound recording. |
+| 'C-line' | 'str' | _Required_. Copyright of the music. |
+| 'language_rating' | 'str' | _Required_. Rates the song. |
+
+#### GET /song/:song_id/comments
+Get all the comments for specific song id. Return array of JSON object.
+
+**Parameter**
+
+| Name  | Type | Description |
+| ----- | ---- | ----------- |
+| 'song_id' | 'int' | _Required_. Identifier for the song. |
+
+
+**Response**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| 'comment_id' | 'int' | _Required_. Identifier for comment. |
+| 'song_id' | 'int' | _Required_. Identifier for which song this comment belongs to. |
+| 'user_id' | 'int' | _Required_. Identify which user wrote the comment. |
+| 'content' | 'str' | _Required_. Comment content. |
+| 'time' | 'int' | _Required_. Timestamp for comment. |
+| 'track_time' | 'int' | _Reuiqred_. Track time stamp for comment. |
+
+### DELETE
+
+
+#### DELETE /song/:song_id
+delete specific song.
+
+**Parameter**
+
+| Name  | Type | Description |
+| ----- | ---- | ----------- |
+| 'song_id' | 'int' | _Required_. Identifier for the song. |
+
+#### DELETE /song/:song_id/comments
+delete all comments from specific song.
+
+**Parameter**
+
+| Name  | Type | Description |
+| ----- | ---- | ----------- |
+| 'song_id' | 'int' | _Required_. Identifier for the song. |
+
+#### DELETE /song/:song_id/comments/:comment_id/
+delete specific comment from specific song.
+
+**Parameter**
+
+| Name  | Type | Description |
+| ----- | ---- | ----------- |
+| 'comment_id' | 'int' | _Required_. Identifier for the comment. |
+
+
+### PUT
+#### PUT /song/:song_id'
+update specific song.
+JSON object goes in body.
+
+**Parameter**
+
+| Name  | Type | Description |
+| ----- | ---- | ----------- |
+| 'song_id' | 'int' | _Required_. Identifier for the comment. |
+
+
+**Body**
+
+| Name | Type | Description |
+| ---- | ---- | ----------- |
+| 'song_id' | 'int' | _Optional_. Identifier for song. |
```

Comment:
If it's a put instead of a patch, shouldn't all the same fields be required? Or do you mean that any of these fields can be changed? (even the ID?)